demo/Titanic/README.md

```diff
@@ -45,6 +45,17 @@ To run the demo execute the `run_demo.py` script in either of the two available
 
 If you choose the `hit` scenario, don't be surprised if you encounter some failures along the route...
 
+### Step 4: Running the demo game (optional)
+
+The demo also contains a playable version of the software under test. To start the game, follow the following instructions
+
+* Install matplotlib: `pip install matplotlib`
+* Based on your system, install curses
+  * for Windows: `pip install windows-curses`
```

Comment:
Would this Windows/Unix difference be resolvable from a `requirements.txt` file?

Reply:
I don't think so, moreover, this is an optional/bonus so putting it in requirements might lead to installing more dependencies than some people might expect/need.

Reply:
We could for different purposes/platforms create different `requirements.txt` files (e.g. `requirements-windows-all.txt` and `requirements-unix-testonly.txt` )